doc/en/example/markers.rst

```diff
@@ -364,7 +364,7 @@ specifies via named environments:
     import pytest
 
 
-    def pytest_addoption(parser):
+    def pytest_addoption(parser, pluginmanager):
```

Comment:
its not  necessary to add the parameters where they arent used

Reply:
Just thought it might be confusing for the reader with the change. I can revert.

Reply:
Removed